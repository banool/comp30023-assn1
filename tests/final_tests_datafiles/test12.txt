0 966 353 3
4 547 286 10
8 67 360 2
9 804 488 1
18 332 436 1
19 184 190 8
27 227 318 3
38 646 293 5
38 893 625 6
40 127 401 9
40 490 453 2
52 835 431 3
64 55 304 1
70 353 243 9
73 775 291 11
77 745 411 5
78 92 283 2
78 557 212 3
78 688 547 8
79 502 568 6
83 418 327 1
85 424 290 2
96 592 532 4
99 260 367 2
99 729 588 4
99 770 212 4
110 663 289 4
110 798 387 3
114 633 471 5
119 323 339 1
121 114 318 3
128 853 443 4
136 561 307 6
139 755 152 3
140 878 208 5
142 881 351 6
147 98 471 4
149 589 317 13
152 536 265 2
161 743 316 4
164 477 390 3
168 648 619 5
177 928 221 3
181 910 447 6
183 513 346 4
189 377 405 2
195 19 372 5
197 340 236 9
198 333 476 4
206 678 369 2
209 30 334 9
209 216 407 1
213 583 433 9
213 713 409 3
215 528 303 16
223 991 309 3
235 961 240 2
243 953 456 13
253 959 391 3
256 163 303 5
257 343 325 9
258 872 314 12
261 145 204 4
263 540 319 4
266 169 562 9
267 888 629 3
285 958 364 3
286 230 347 1
292 223 654 5
299 823 434 9
307 788 373 8
307 887 272 3
316 125 284 1
318 15 259 7
318 235 249 12
318 430 271 3
324 140 349 4
324 390 350 10
333 529 478 4
337 558 550 11
340 262 456 1
348 124 192 6
348 142 340 6
348 653 237 1
350 683 399 4
354 546 193 2
356 747 358 19
358 723 456 11
361 168 389 2
364 485 300 3
365 945 294 6
368 136 226 3
376 5 278 10
377 122 463 3
377 911 262 8
383 363 168 4
385 126 262 11
386 351 405 3
389 370 261 7
390 304 246 1
394 91 440 7
399 531 288 3
404 76 313 4
408 481 153 5
409 450 486 1
412 890 385 4
413 307 228 4
415 17 465 1
415 749 516 3
417 701 527 2
420 256 312 2
428 643 491 8
434 541 256 1
445 675 162 9
445 767 651 4
451 111 203 7
459 80 369 3
459 429 371 2
462 608 470 3
464 660 274 3
471 543 367 2
474 765 623 2
485 869 528 2
485 903 266 5
490 462 315 4
492 183 312 7
492 395 409 9
494 387 493 1
502 159 372 3
505 940 407 6
508 23 370 2
517 498 245 4
519 687 256 4
529 740 336 5
533 826 411 2
536 34 454 2
543 925 210 1
545 415 186 2
550 53 324 1
550 801 149 10
551 891 384 8
554 971 195 2
561 595 482 3
567 423 344 1
572 38 416 5
576 964 480 4
584 930 201 8
589 56 243 3
591 752 197 3
593 326 92 7
596 282 205 1
600 699 489 7
605 703 413 5
606 665 597 9
615 681 340 2
618 45 383 1
622 902 398 5
629 777 181 1
630 885 218 7
630 921 391 4
632 233 411 6
635 298 326 5
638 710 427 4
640 24 362 7
643 95 338 6
643 276 277 6
645 63 197 10
652 922 637 1
654 243 382 12
657 201 589 1
663 1 471 4
663 101 127 6
669 162 249 2
671 189 425 1
674 118 226 3
675 8 214 9
682 204 248 1
688 360 477 5
688 799 324 5
694 476 318 9
708 431 285 10
711 521 473 6
712 766 355 5
715 680 307 2
716 303 140 3
718 134 360 2
730 508 462 5
741 594 470 10
742 270 246 4
743 12 138 4
748 832 254 2
754 950 316 3
757 99 327 3
759 623 295 7
765 931 512 3
776 228 215 3
780 661 333 8
781 937 559 1
784 813 211 2
786 287 242 1
