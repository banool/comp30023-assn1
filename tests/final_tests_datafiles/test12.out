time 0, 966 running, numprocesses=1, numholes=1, memusage=36%
time 2, 966 running, numprocesses=1, numholes=1, memusage=36%
time 4, 547 running, numprocesses=1, numholes=1, memusage=29%
time 6, 547 running, numprocesses=1, numholes=1, memusage=29%
time 10, 67 running, numprocesses=2, numholes=1, memusage=65%
time 12, 804 running, numprocesses=2, numholes=1, memusage=78%
time 13, 547 running, numprocesses=1, numholes=1, memusage=29%
time 18, 332 running, numprocesses=1, numholes=1, memusage=44%
time 19, 184 running, numprocesses=1, numholes=1, memusage=19%
time 21, 184 running, numprocesses=1, numholes=1, memusage=19%
time 25, 184 running, numprocesses=1, numholes=1, memusage=19%
time 27, 227 running, numprocesses=1, numholes=1, memusage=32%
time 29, 227 running, numprocesses=1, numholes=1, memusage=32%
time 38, 646 running, numprocesses=1, numholes=1, memusage=30%
time 40, 893 running, numprocesses=2, numholes=1, memusage=92%
time 42, 127 running, numprocesses=2, numholes=1, memusage=70%
time 44, 490 running, numprocesses=2, numholes=1, memusage=75%
time 46, 646 running, numprocesses=1, numholes=1, memusage=30%
time 49, 893 running, numprocesses=1, numholes=1, memusage=63%
time 53, 835 running, numprocesses=1, numholes=1, memusage=44%
time 55, 127 running, numprocesses=2, numholes=1, memusage=84%
time 59, 835 running, numprocesses=2, numholes=1, memusage=84%
time 60, 127 running, numprocesses=1, numholes=2, memusage=41%
time 64, 55 running, numprocesses=1, numholes=1, memusage=31%
time 70, 353 running, numprocesses=1, numholes=1, memusage=25%
time 72, 353 running, numprocesses=1, numholes=1, memusage=25%
time 76, 775 running, numprocesses=2, numholes=1, memusage=54%
time 78, 745 running, numprocesses=3, numholes=1, memusage=95%
time 80, 92 running, numprocesses=3, numholes=1, memusage=82%
time 82, 557 running, numprocesses=3, numholes=1, memusage=75%
time 84, 688 running, numprocesses=1, numholes=1, memusage=55%
time 86, 502 running, numprocesses=1, numholes=1, memusage=57%
time 88, 418 running, numprocesses=2, numholes=1, memusage=90%
time 89, 424 running, numprocesses=2, numholes=1, memusage=86%
time 91, 775 running, numprocesses=2, numholes=1, memusage=86%
time 95, 745 running, numprocesses=2, numholes=2, memusage=71%
time 98, 592 running, numprocesses=2, numholes=2, memusage=83%
time 100, 260 running, numprocesses=2, numholes=2, memusage=66%
time 102, 729 running, numprocesses=1, numholes=1, memusage=59%
time 104, 770 running, numprocesses=2, numholes=1, memusage=80%
time 106, 557 running, numprocesses=2, numholes=2, memusage=43%
time 107, 688 running, numprocesses=2, numholes=2, memusage=76%
time 111, 663 running, numprocesses=2, numholes=2, memusage=51%
time 113, 798 running, numprocesses=2, numholes=2, memusage=60%
time 115, 633 running, numprocesses=2, numholes=2, memusage=69%
time 117, 502 running, numprocesses=2, numholes=2, memusage=78%
time 121, 323 running, numprocesses=2, numholes=2, memusage=56%
time 122, 114 running, numprocesses=2, numholes=2, memusage=53%
time 124, 592 running, numprocesses=2, numholes=2, memusage=75%
time 126, 729 running, numprocesses=2, numholes=1, memusage=80%
time 128, 853 running, numprocesses=2, numholes=2, memusage=66%
time 130, 770 running, numprocesses=2, numholes=2, memusage=66%
time 132, 663 running, numprocesses=2, numholes=1, memusage=74%
time 134, 798 running, numprocesses=2, numholes=1, memusage=83%
time 135, 633 running, numprocesses=2, numholes=1, memusage=92%
time 138, 561 running, numprocesses=2, numholes=1, memusage=75%
time 140, 755 running, numprocesses=3, numholes=1, memusage=91%
time 142, 878 running, numprocesses=3, numholes=2, memusage=67%
time 144, 881 running, numprocesses=3, numholes=2, memusage=72%
time 146, 114 running, numprocesses=3, numholes=2, memusage=68%
time 147, 98 running, numprocesses=3, numholes=2, memusage=84%
time 149, 589 running, numprocesses=3, numholes=2, memusage=68%
time 151, 853 running, numprocesses=3, numholes=2, memusage=81%
time 153, 536 running, numprocesses=3, numholes=2, memusage=63%
time 155, 561 running, numprocesses=3, numholes=2, memusage=67%
time 159, 755 running, numprocesses=2, numholes=2, memusage=36%
time 160, 878 running, numprocesses=1, numholes=1, memusage=21%
time 163, 743 running, numprocesses=1, numholes=1, memusage=32%
time 165, 477 running, numprocesses=2, numholes=1, memusage=71%
time 167, 881 running, numprocesses=2, numholes=1, memusage=67%
time 171, 648 running, numprocesses=2, numholes=1, memusage=94%
time 173, 98 running, numprocesses=2, numholes=1, memusage=79%
time 175, 589 running, numprocesses=2, numholes=1, memusage=64%
time 179, 928 running, numprocesses=3, numholes=1, memusage=86%
time 181, 910 running, numprocesses=2, numholes=2, memusage=67%
time 183, 513 running, numprocesses=2, numholes=2, memusage=57%
time 185, 743 running, numprocesses=2, numholes=2, memusage=54%
time 187, 477 running, numprocesses=2, numholes=2, memusage=62%
time 188, 648 running, numprocesses=2, numholes=2, memusage=84%
time 191, 377 running, numprocesses=2, numholes=2, memusage=63%
time 193, 928 running, numprocesses=1, numholes=2, memusage=23%
time 194, 910 running, numprocesses=1, numholes=1, memusage=45%
time 198, 19 running, numprocesses=1, numholes=1, memusage=38%
time 200, 340 running, numprocesses=2, numholes=1, memusage=61%
time 202, 333 running, numprocesses=1, numholes=1, memusage=48%
time 204, 513 running, numprocesses=2, numholes=1, memusage=83%
time 206, 678 running, numprocesses=2, numholes=1, memusage=85%
time 208, 19 running, numprocesses=2, numholes=1, memusage=85%
time 211, 30 running, numprocesses=2, numholes=1, memusage=81%
time 213, 216 running, numprocesses=2, numholes=2, memusage=75%
time 214, 583 running, numprocesses=2, numholes=2, memusage=77%
time 216, 713 running, numprocesses=2, numholes=2, memusage=75%
time 218, 528 running, numprocesses=2, numholes=2, memusage=64%
time 220, 340 running, numprocesses=2, numholes=1, memusage=54%
time 224, 991 running, numprocesses=3, numholes=1, memusage=85%
time 226, 333 running, numprocesses=1, numholes=1, memusage=48%
time 228, 30 running, numprocesses=1, numholes=1, memusage=34%
time 232, 583 running, numprocesses=2, numholes=1, memusage=77%
time 236, 961 running, numprocesses=2, numholes=1, memusage=58%
time 238, 713 running, numprocesses=2, numholes=1, memusage=75%
time 239, 528 running, numprocesses=2, numholes=1, memusage=64%
time 243, 953 running, numprocesses=1, numholes=1, memusage=46%
time 245, 991 running, numprocesses=2, numholes=1, memusage=77%
time 246, 953 running, numprocesses=1, numholes=1, memusage=46%
time 250, 353 running, numprocesses=2, numholes=1, memusage=70%
time 253, 959 running, numprocesses=2, numholes=1, memusage=85%
time 255, 959 running, numprocesses=2, numholes=1, memusage=85%
time 256, 163 running, numprocesses=2, numholes=1, memusage=76%
time 258, 343 running, numprocesses=2, numholes=2, memusage=63%
time 260, 872 running, numprocesses=2, numholes=2, memusage=62%
time 262, 145 running, numprocesses=3, numholes=2, memusage=83%
time 264, 540 running, numprocesses=3, numholes=2, memusage=83%
time 266, 169 running, numprocesses=2, numholes=2, memusage=77%
time 268, 888 running, numprocesses=2, numholes=2, memusage=84%
time 270, 163 running, numprocesses=2, numholes=2, memusage=51%
time 273, 343 running, numprocesses=2, numholes=2, memusage=53%
time 277, 872 running, numprocesses=3, numholes=2, memusage=85%
time 281, 145 running, numprocesses=3, numholes=2, memusage=85%
time 283, 540 running, numprocesses=3, numholes=1, memusage=96%
time 285, 958 running, numprocesses=1, numholes=1, memusage=37%
time 287, 230 running, numprocesses=2, numholes=1, memusage=72%
time 288, 169 running, numprocesses=2, numholes=1, memusage=93%
time 292, 223 running, numprocesses=1, numholes=1, memusage=66%
time 294, 888 running, numprocesses=1, numholes=1, memusage=63%
time 295, 958 running, numprocesses=1, numholes=1, memusage=37%
time 296, 223 running, numprocesses=1, numholes=1, memusage=66%
time 299, 823 running, numprocesses=1, numholes=1, memusage=44%
time 301, 823 running, numprocesses=1, numholes=1, memusage=44%
time 305, 775 running, numprocesses=2, numholes=1, memusage=73%
time 310, 788 running, numprocesses=2, numholes=1, memusage=81%
time 312, 887 running, numprocesses=2, numholes=2, memusage=65%
time 314, 788 running, numprocesses=2, numholes=2, memusage=65%
time 318, 125 running, numprocesses=2, numholes=1, memusage=56%
time 319, 15 running, numprocesses=2, numholes=1, memusage=54%
time 321, 235 running, numprocesses=3, numholes=1, memusage=78%
time 323, 430 running, numprocesses=3, numholes=2, memusage=78%
time 325, 140 running, numprocesses=2, numholes=2, memusage=60%
time 327, 390 running, numprocesses=2, numholes=2, memusage=60%
time 329, 887 running, numprocesses=2, numholes=2, memusage=53%
time 330, 15 running, numprocesses=2, numholes=2, memusage=51%
time 334, 529 running, numprocesses=2, numholes=2, memusage=73%
time 336, 235 running, numprocesses=2, numholes=2, memusage=73%
time 340, 558 running, numprocesses=1, numholes=1, memusage=55%
time 342, 262 running, numprocesses=1, numholes=1, memusage=46%
time 343, 430 running, numprocesses=1, numholes=1, memusage=28%
time 344, 140 running, numprocesses=1, numholes=1, memusage=35%
time 346, 390 running, numprocesses=1, numholes=1, memusage=35%
time 350, 124 running, numprocesses=2, numholes=1, memusage=55%
time 352, 142 running, numprocesses=3, numholes=1, memusage=89%
time 354, 653 running, numprocesses=3, numholes=2, memusage=77%
time 355, 683 running, numprocesses=2, numholes=2, memusage=60%
time 357, 546 running, numprocesses=3, numholes=2, memusage=79%
time 359, 747 running, numprocesses=2, numholes=2, memusage=55%
time 361, 723 running, numprocesses=2, numholes=2, memusage=65%
time 363, 168 running, numprocesses=2, numholes=2, memusage=59%
time 365, 485 running, numprocesses=2, numholes=2, memusage=50%
time 367, 945 running, numprocesses=3, numholes=2, memusage=79%
time 369, 136 running, numprocesses=3, numholes=2, memusage=72%
time 371, 529 running, numprocesses=1, numholes=1, memusage=48%
time 373, 558 running, numprocesses=1, numholes=1, memusage=55%
time 377, 5 running, numprocesses=2, numholes=1, memusage=83%
time 379, 122 running, numprocesses=2, numholes=2, memusage=75%
time 381, 911 running, numprocesses=2, numholes=2, memusage=54%
time 383, 363 running, numprocesses=3, numholes=2, memusage=71%
time 385, 126 running, numprocesses=3, numholes=1, memusage=70%
time 387, 351 running, numprocesses=2, numholes=2, memusage=58%
time 389, 370 running, numprocesses=3, numholes=2, memusage=84%
time 391, 304 running, numprocesses=3, numholes=2, memusage=68%
time 392, 124 running, numprocesses=3, numholes=2, memusage=63%
time 396, 91 running, numprocesses=3, numholes=2, memusage=87%
time 398, 142 running, numprocesses=3, numholes=2, memusage=77%
time 402, 531 running, numprocesses=3, numholes=2, memusage=72%
time 404, 76 running, numprocesses=3, numholes=2, memusage=75%
time 406, 683 running, numprocesses=3, numholes=2, memusage=83%
time 408, 481 running, numprocesses=3, numholes=2, memusage=59%
time 410, 450 running, numprocesses=1, numholes=1, memusage=49%
time 411, 747 running, numprocesses=1, numholes=1, memusage=36%
time 415, 890 running, numprocesses=2, numholes=1, memusage=75%
time 417, 307 running, numprocesses=3, numholes=1, memusage=98%
time 419, 17 running, numprocesses=2, numholes=2, memusage=70%
time 420, 749 running, numprocesses=2, numholes=2, memusage=75%
time 422, 701 running, numprocesses=2, numholes=2, memusage=76%
time 424, 256 running, numprocesses=2, numholes=2, memusage=54%
time 426, 723 running, numprocesses=2, numholes=2, memusage=69%
time 430, 643 running, numprocesses=2, numholes=2, memusage=72%
time 432, 485 running, numprocesses=2, numholes=2, memusage=53%
time 433, 945 running, numprocesses=2, numholes=2, memusage=53%
time 437, 541 running, numprocesses=2, numholes=2, memusage=49%
time 438, 136 running, numprocesses=2, numholes=2, memusage=46%
time 439, 5 running, numprocesses=2, numholes=2, memusage=51%
time 443, 122 running, numprocesses=3, numholes=2, memusage=97%
time 444, 911 running, numprocesses=3, numholes=2, memusage=77%
time 448, 675 running, numprocesses=4, numholes=2, memusage=93%
time 450, 767 running, numprocesses=1, numholes=1, memusage=66%
time 452, 111 running, numprocesses=2, numholes=1, memusage=86%
time 454, 363 running, numprocesses=2, numholes=2, memusage=38%
time 456, 126 running, numprocesses=2, numholes=2, memusage=47%
time 460, 80 running, numprocesses=3, numholes=2, memusage=84%
time 462, 429 running, numprocesses=3, numholes=2, memusage=84%
time 464, 608 running, numprocesses=2, numholes=2, memusage=68%
time 466, 660 running, numprocesses=2, numholes=2, memusage=48%
time 468, 351 running, numprocesses=2, numholes=2, memusage=61%
time 469, 370 running, numprocesses=2, numholes=2, memusage=47%
time 473, 543 running, numprocesses=3, numholes=2, memusage=84%
time 475, 765 running, numprocesses=2, numholes=2, memusage=83%
time 477, 91 running, numprocesses=2, numholes=2, memusage=65%
time 481, 531 running, numprocesses=2, numholes=2, memusage=50%
time 482, 76 running, numprocesses=2, numholes=2, memusage=52%
time 484, 481 running, numprocesses=2, numholes=2, memusage=36%
time 487, 869 running, numprocesses=2, numholes=2, memusage=74%
time 489, 903 running, numprocesses=2, numholes=2, memusage=47%
time 491, 462 running, numprocesses=3, numholes=2, memusage=79%
time 493, 183 running, numprocesses=3, numholes=2, memusage=79%
time 495, 395 running, numprocesses=2, numholes=2, memusage=62%
time 497, 387 running, numprocesses=2, numholes=2, memusage=70%
time 498, 890 running, numprocesses=2, numholes=2, memusage=59%
time 500, 307 running, numprocesses=2, numholes=2, memusage=44%
time 502, 159 running, numprocesses=2, numholes=2, memusage=58%
time 504, 749 running, numprocesses=2, numholes=2, memusage=72%
time 505, 940 running, numprocesses=2, numholes=2, memusage=61%
time 507, 643 running, numprocesses=2, numholes=2, memusage=70%
time 511, 23 running, numprocesses=2, numholes=2, memusage=58%
time 513, 675 running, numprocesses=2, numholes=2, memusage=37%
time 517, 498 running, numprocesses=3, numholes=2, memusage=61%
time 519, 687 running, numprocesses=3, numholes=2, memusage=63%
time 521, 767 running, numprocesses=2, numholes=1, memusage=82%
time 523, 111 running, numprocesses=2, numholes=1, memusage=37%
time 527, 80 running, numprocesses=3, numholes=1, memusage=74%
time 528, 608 running, numprocesses=3, numholes=1, memusage=84%
time 529, 740 running, numprocesses=3, numholes=1, memusage=71%
time 531, 660 running, numprocesses=4, numholes=1, memusage=98%
time 532, 903 running, numprocesses=4, numholes=1, memusage=97%
time 535, 826 running, numprocesses=3, numholes=1, memusage=78%
time 537, 34 running, numprocesses=3, numholes=1, memusage=82%
time 539, 462 running, numprocesses=3, numholes=1, memusage=68%
time 541, 183 running, numprocesses=3, numholes=1, memusage=68%
time 545, 925 running, numprocesses=4, numholes=1, memusage=89%
time 546, 415 running, numprocesses=4, numholes=1, memusage=87%
time 548, 395 running, numprocesses=3, numholes=1, memusage=78%
time 552, 53 running, numprocesses=3, numholes=1, memusage=69%
time 553, 801 running, numprocesses=3, numholes=1, memusage=52%
time 555, 891 running, numprocesses=4, numholes=1, memusage=90%
time 557, 971 running, numprocesses=4, numholes=1, memusage=71%
time 559, 159 running, numprocesses=4, numholes=1, memusage=89%
time 560, 940 running, numprocesses=4, numholes=1, memusage=93%
time 564, 595 running, numprocesses=4, numholes=1, memusage=100%
time 566, 498 running, numprocesses=4, numholes=1, memusage=76%
time 568, 423 running, numprocesses=4, numholes=1, memusage=86%
time 569, 687 running, numprocesses=4, numholes=1, memusage=77%
time 571, 740 running, numprocesses=4, numholes=1, memusage=85%
time 574, 38 running, numprocesses=4, numholes=1, memusage=93%
time 576, 964 running, numprocesses=4, numholes=1, memusage=100%
time 578, 801 running, numprocesses=4, numholes=1, memusage=100%
time 582, 891 running, numprocesses=4, numholes=1, memusage=90%
time 586, 930 running, numprocesses=4, numholes=1, memusage=72%
time 588, 595 running, numprocesses=3, numholes=2, memusage=80%
time 589, 56 running, numprocesses=3, numholes=2, memusage=56%
time 591, 752 running, numprocesses=4, numholes=2, memusage=76%
time 593, 326 running, numprocesses=5, numholes=2, memusage=85%
time 595, 38 running, numprocesses=2, numholes=2, memusage=51%
time 598, 282 running, numprocesses=2, numholes=2, memusage=30%
time 599, 964 running, numprocesses=2, numholes=2, memusage=58%
time 601, 699 running, numprocesses=2, numholes=2, memusage=59%
time 603, 930 running, numprocesses=3, numholes=2, memusage=79%
time 607, 703 running, numprocesses=3, numholes=3, memusage=71%
time 609, 665 running, numprocesses=2, numholes=2, memusage=69%
time 611, 56 running, numprocesses=2, numholes=2, memusage=34%
time 612, 752 running, numprocesses=2, numholes=2, memusage=29%
time 613, 326 running, numprocesses=1, numholes=2, memusage=10%
time 617, 681 running, numprocesses=2, numholes=2, memusage=44%
time 619, 45 running, numprocesses=2, numholes=2, memusage=48%
time 620, 699 running, numprocesses=2, numholes=2, memusage=59%
time 624, 902 running, numprocesses=2, numholes=2, memusage=49%
time 626, 703 running, numprocesses=2, numholes=2, memusage=51%
time 629, 777 running, numprocesses=2, numholes=2, memusage=28%
time 630, 885 running, numprocesses=2, numholes=2, memusage=31%
time 632, 921 running, numprocesses=3, numholes=2, memusage=71%
time 634, 233 running, numprocesses=3, numholes=2, memusage=73%
time 636, 298 running, numprocesses=3, numholes=2, memusage=64%
time 638, 710 running, numprocesses=3, numholes=2, memusage=74%
time 640, 24 running, numprocesses=3, numholes=2, memusage=68%
time 642, 665 running, numprocesses=2, numholes=2, memusage=69%
time 646, 95 running, numprocesses=2, numholes=2, memusage=43%
time 648, 276 running, numprocesses=3, numholes=2, memusage=71%
time 650, 63 running, numprocesses=3, numholes=3, memusage=57%
time 652, 922 running, numprocesses=2, numholes=2, memusage=73%
time 653, 902 running, numprocesses=2, numholes=2, memusage=49%
time 656, 243 running, numprocesses=2, numholes=2, memusage=48%
time 658, 201 running, numprocesses=2, numholes=2, memusage=69%
time 659, 885 running, numprocesses=2, numholes=2, memusage=31%
time 663, 1 running, numprocesses=3, numholes=2, memusage=79%
time 665, 101 running, numprocesses=4, numholes=2, memusage=91%
time 667, 921 running, numprocesses=4, numholes=2, memusage=83%
time 669, 162 running, numprocesses=4, numholes=2, memusage=69%
time 671, 189 running, numprocesses=4, numholes=2, memusage=87%
time 672, 233 running, numprocesses=4, numholes=2, memusage=85%
time 676, 118 running, numprocesses=4, numholes=2, memusage=67%
time 678, 8 running, numprocesses=5, numholes=2, memusage=88%
time 680, 298 running, numprocesses=4, numholes=3, memusage=76%
time 683, 204 running, numprocesses=4, numholes=3, memusage=69%
time 684, 710 running, numprocesses=4, numholes=3, memusage=86%
time 686, 24 running, numprocesses=4, numholes=3, memusage=80%
time 690, 360 running, numprocesses=3, numholes=2, memusage=70%
time 692, 799 running, numprocesses=3, numholes=2, memusage=55%
time 694, 476 running, numprocesses=4, numholes=2, memusage=87%
time 696, 95 running, numprocesses=3, numholes=2, memusage=56%
time 700, 276 running, numprocesses=3, numholes=2, memusage=50%
time 704, 63 running, numprocesses=3, numholes=2, memusage=42%
time 708, 431 running, numprocesses=4, numholes=2, memusage=71%
time 710, 243 running, numprocesses=4, numholes=2, memusage=80%
time 714, 521 running, numprocesses=4, numholes=2, memusage=89%
time 716, 766 running, numprocesses=4, numholes=2, memusage=78%
time 718, 680 running, numprocesses=4, numholes=2, memusage=73%
time 720, 303 running, numprocesses=4, numholes=2, memusage=56%
time 722, 134 running, numprocesses=5, numholes=2, memusage=92%
time 724, 1 running, numprocesses=3, numholes=2, memusage=69%
time 726, 101 running, numprocesses=2, numholes=2, memusage=22%
time 730, 508 running, numprocesses=2, numholes=2, memusage=56%
time 732, 118 running, numprocesses=3, numholes=2, memusage=78%
time 733, 8 running, numprocesses=3, numholes=2, memusage=77%
time 737, 360 running, numprocesses=2, numholes=2, memusage=57%
time 740, 799 running, numprocesses=2, numholes=2, memusage=42%
time 743, 594 running, numprocesses=2, numholes=2, memusage=57%
time 745, 270 running, numprocesses=3, numholes=2, memusage=81%
time 747, 12 running, numprocesses=4, numholes=2, memusage=95%
time 749, 832 running, numprocesses=4, numholes=3, memusage=73%
time 751, 476 running, numprocesses=4, numholes=3, memusage=80%
time 755, 950 running, numprocesses=4, numholes=3, memusage=80%
time 757, 99 running, numprocesses=4, numholes=3, memusage=81%
time 759, 623 running, numprocesses=4, numholes=3, memusage=78%
time 761, 431 running, numprocesses=4, numholes=3, memusage=77%
time 765, 931 running, numprocesses=3, numholes=2, memusage=75%
time 767, 521 running, numprocesses=3, numholes=2, memusage=71%
time 771, 766 running, numprocesses=3, numholes=2, memusage=59%
time 774, 303 running, numprocesses=3, numholes=2, memusage=37%
time 775, 508 running, numprocesses=3, numholes=2, memusage=70%
time 778, 228 running, numprocesses=3, numholes=2, memusage=45%
time 780, 661 running, numprocesses=4, numholes=2, memusage=78%
time 782, 937 running, numprocesses=3, numholes=2, memusage=79%
time 783, 594 running, numprocesses=3, numholes=2, memusage=70%
time 787, 813 running, numprocesses=4, numholes=2, memusage=92%
time 789, 287 running, numprocesses=4, numholes=2, memusage=95%
time 790, 270 running, numprocesses=4, numholes=2, memusage=95%
time 792, 12 running, numprocesses=3, numholes=2, memusage=70%
time 794, 950 running, numprocesses=2, numholes=2, memusage=41%
time 795, 99 running, numprocesses=2, numholes=2, memusage=42%
time 796, 623 running, numprocesses=2, numholes=2, memusage=39%
time 800, 931 running, numprocesses=2, numholes=2, memusage=61%
time 801, 228 running, numprocesses=2, numholes=2, memusage=31%
time 802, 661 running, numprocesses=2, numholes=2, memusage=43%
time 806, 688 running, numprocesses=2, numholes=2, memusage=64%
time 808, 589 running, numprocesses=2, numholes=2, memusage=41%
time 815, 340 running, numprocesses=2, numholes=2, memusage=33%
time 818, 30 running, numprocesses=2, numholes=2, memusage=43%
time 821, 583 running, numprocesses=2, numholes=2, memusage=53%
time 824, 528 running, numprocesses=2, numholes=2, memusage=40%
time 832, 953 running, numprocesses=2, numholes=2, memusage=55%
time 839, 343 running, numprocesses=2, numholes=2, memusage=42%
time 842, 872 running, numprocesses=2, numholes=2, memusage=41%
time 848, 169 running, numprocesses=2, numholes=2, memusage=66%
time 851, 823 running, numprocesses=2, numholes=2, memusage=53%
time 854, 788 running, numprocesses=2, numholes=2, memusage=47%
time 856, 15 running, numprocesses=2, numholes=2, memusage=36%
time 857, 235 running, numprocesses=2, numholes=2, memusage=35%
time 863, 390 running, numprocesses=2, numholes=2, memusage=45%
time 867, 558 running, numprocesses=2, numholes=2, memusage=65%
time 872, 747 running, numprocesses=2, numholes=2, memusage=45%
time 880, 723 running, numprocesses=2, numholes=2, memusage=55%
time 885, 5 running, numprocesses=2, numholes=2, memusage=37%
time 889, 911 running, numprocesses=2, numholes=2, memusage=36%
time 891, 126 running, numprocesses=2, numholes=2, memusage=36%
time 896, 370 running, numprocesses=2, numholes=2, memusage=36%
time 897, 91 running, numprocesses=2, numholes=2, memusage=54%
time 898, 643 running, numprocesses=2, numholes=2, memusage=59%
time 900, 675 running, numprocesses=2, numholes=2, memusage=26%
time 903, 111 running, numprocesses=2, numholes=2, memusage=30%
time 904, 183 running, numprocesses=2, numholes=2, memusage=41%
time 905, 395 running, numprocesses=2, numholes=2, memusage=51%
time 908, 801 running, numprocesses=2, numholes=2, memusage=25%
time 912, 891 running, numprocesses=2, numholes=2, memusage=48%
time 914, 930 running, numprocesses=2, numholes=2, memusage=30%
time 916, 326 running, numprocesses=1, numholes=2, memusage=10%
time 917, 699 running, numprocesses=1, numholes=1, memusage=49%
time 918, 665 running, numprocesses=1, numholes=1, memusage=60%
time 921, 885 running, numprocesses=1, numholes=1, memusage=22%
time 922, 24 running, numprocesses=1, numholes=1, memusage=37%
time 923, 63 running, numprocesses=1, numholes=1, memusage=20%
time 927, 243 running, numprocesses=1, numholes=1, memusage=39%
time 933, 8 running, numprocesses=1, numholes=1, memusage=22%
time 936, 476 running, numprocesses=1, numholes=1, memusage=32%
time 939, 431 running, numprocesses=1, numholes=1, memusage=29%
time 943, 594 running, numprocesses=1, numholes=1, memusage=47%
time 947, 623 running, numprocesses=1, numholes=1, memusage=30%
time 948, 661 running, numprocesses=1, numholes=1, memusage=34%
time 950, 528 running, numprocesses=1, numholes=1, memusage=31%
time 952, 747 running, numprocesses=1, numholes=1, memusage=36%
time 957, simulation finished.
